CONTRIBUTING.md

```diff
@@ -4,19 +4,21 @@
 
 ### Installing
 
-Once the repo has been cloned, install Lerna:
+Once the repo has been cloned, install Lerna
 
 ```bash
 yarn global add lerna
 ```
 
-Install all package dependencies:
+Install all package dependencies and build all workspaces (local dependencies)
 
 ```bash
-yarn install
+yarn
 ```
 
-Build the workspaces (local dependencies)
+### Maintaining
+
+Build all workspaces (local dependencies)
```

Comment:
Should we add more information as to when this should be run?

Reply:
You can submit your suggestion as part of this comment or in a new PR.  This PR was only intended to streamline and remove unnecessary documentation.